hydroshare/tasks.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import absolute_import
+
+from celery import Celery
+from django.conf import settings
+
```

Comment:
RabbitMQ was being used only for Celery. Use redis instead to consolidate, allowing us to remove RabbitMQ entirely from the project.